cvat/apps/engine/background.py

```diff
@@ -91,15 +91,17 @@ def handle_existing_rq_job(
 
         rq_job_status = rq_job.get_status(refresh=False)
 
-        if rq_job_status in {RQJobStatus.STARTED, RQJobStatus.QUEUED}:
+        if rq_job_status in {
+            RQJobStatus.STARTED,
+            RQJobStatus.QUEUED,
+            RQJobStatus.DEFERRED,
+        }:
             return Response(
-                data="Export request is being processed",
+                RqIdSerializer({"rq_id": rq_job.id}).data,
```

Comment:
Doing so for `started` jobs is probably not entirely correct, as the export results in this case might be outdated. I don't think it's a big issue so far to be fixed immediately, but it makes sense to add a comment about this decision. Our export implementation doesn't lock the whole object during the export, so there are multiple ways to get outdated exported data.